sormas-app/app/src/main/java/de/symeda/sormas/app/backend/environment/Environment.java

```diff
@@ -195,22 +196,25 @@ public void setWaterUseJson(String waterUseJson) {
 	}
 
 	public Map<WaterUse, Boolean> getWateruse() {
-        if (wateruse == null) {
-            Gson gson = new Gson();
-            Type type = new TypeToken<Map<WaterUse, Boolean>>() {
-            }.getType();
-            wateruse = gson.fromJson(waterUseJson, type);
-            if (wateruse == null) {
-                wateruse = new HashMap<>();
-            }
-        }
+		if (wateruse == null) {
+			Gson gson = new GsonBuilder().enableComplexMapKeySerialization().registerTypeAdapter(WaterUse.class, new WaterUseSerializer()).create();
```

Comment:
please introduce a method for creating the Gson instance

Reply:
created